### ISSUE TITLE
Progress Bar not showing up

### ISSUE THREAD

**Issue by ShivamSharma2705** [microsoft/vscode-jupyter#11014]
I have been using VS code for a while now and all of a sudden the progress bar stops appearing. 

I am using 

from tqdm.notebook import tqdm

![Screen Shot 2022-08-01 at 10 28 59 PM](https://user-images.githubusercontent.com/94197666/182298737-228e2305-487b-4411-9fa2-db841068fec7.png)
.

**Comment by rchiodo** [comment]
Thanks for the issue. 

What's your version of VS code and version of the Jupyter extension?

I cannot repro with the latest of each:

![image](https://user-images.githubusercontent.com/19672699/182424254-0544895d-26d7-4382-8db0-34c56c583a7a.png)

**Comment by ShivamSharma2705** [comment]
Hey @rchiodo 

Thanks for the quick follow up!

VS Code Version = 1.67.0 (Universal)
Jupyter Extension version = v2022.5.1001311010

**Comment by rchiodo** [comment]
Your version of VS code and Jupyter are a couple months out of date but it's weird that it would suddenly stop working. Did you update tqdm?

Latest VS code is 1.69.2. Latest Jupyter that matches would be 2022.6.1201981810

You might try updating VS code and Jupyter but I suspect something else changed.

**Comment by kurazu** [comment]
I also experience this bug. I haven't been using the extension previously, so can't say if it was working previously.
![image](https://user-images.githubusercontent.com/903740/185925970-7464ad4e-5ee7-49d5-9e8f-f9f092311497.png)

VS Code Version:
```
Version: 1.70.2
Commit: e4503b30fc78200f846c62cf8091b76ff5547662
Date: 2022-08-16T05:36:25.715Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Linux x64 5.4.0-122-generic
```

Jupyter extension: `v2022.7.1102252217`

Python: `3.8.10`

Relevant installed packages:
```sh
$ pip freeze | grep -E '(jupyter|tqdm)'
jupyter==1.0.0
jupyter-client==7.3.4
jupyter-console==6.4.4
jupyter-core==4.11.1
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.2
tqdm==4.64.0
```

**Comment by DonJayamanne** [comment]
@kurazu 
Please could you:
* Capture all of the output from the `Jupyter` output panel (you can get there using the command `Jupyter: Show Output` from the command palette), 
    * please copy everything and upload that here
    * Please ensure you first run the cell with tqdm
* Once you have executed the cell, please could you upload the ipynb file as well, I'd like to see the contents of the ipynb file (with the output)

**Comment by kurazu** [comment]
@DonJayamanne here are the requested files:
[jupyter_output.log](https://github.com/microsoft/vscode-jupyter/files/9396830/jupyter_output.log)
[test.ipynb.txt](https://github.com/microsoft/vscode-jupyter/files/9396837/test.ipynb.txt)

For completeness: the same notebook does work fine in plain `jupyter notebook` through the browser: 
![image](https://user-images.githubusercontent.com/903740/185997057-fd4e15fb-0bfd-430b-bbef-69ab942b20ae.png)

**Comment by DonJayamanne** [comment]
@kurazu can you run the following:
`pip freeze | grep -E '(jupyter|tqdm|ipy)'

I'd like to see versions of ipywidgets, ipykernel, ipython

**Comment by kurazu** [comment]
@DonJayamanne here you go:
```
ipykernel==6.15.1
ipython==8.4.0
ipython-genutils==0.2.0
ipywidgets==8.0.1
jupyter==1.0.0
jupyter-client==7.3.4
jupyter-console==6.4.4
jupyter-core==4.11.1
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.2
tqdm==4.64.0
```

**Comment by carlthome** [comment]
Same problem here. Used to see them, but not anymore.

**Comment by carlthome** [comment]
```sh
pip freeze | grep -E '(jupyter|tqdm|ipy)

ipykernel==6.15.1
ipython==8.4.0
ipython-genutils==0.2.0
ipywidgets==8.0.1
jupyter==1.0.0
jupyter-client==7.3.4
jupyter-console==6.4.4
jupyter-core==4.11.1
jupyter-server==1.18.1
jupyter-server-mathjax==0.2.6
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.2
scipy==1.9.0
tqdm==4.64.0
```

**Comment by DonJayamanne** [comment]
Hi @kurazu @carlthome 
I'm sorry about this bug, 
# this is a caused by Jupyter extension not supporting the latest version of IPYWidgets 8.0.
# As a work around, please downgrade IPYWidgets to 7.xx

Duplicate of #8552

**Comment by DonJayamanne** [comment]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/8552

**Comment by kurazu** [comment]
@DonJayamanne Thank you!

I confirm that pinning `ipywidgets>=7.0,<8.0` solves the issue.

**Comment by trueToastedCode** [comment]
> I still have the issue with :
> 
> VS code 1.71.0 Jupyter extension v2022.8.1002431955
> 
> ```
> conda list | grep -E '(jupyter|tqdm|widget)'
> ipywidgets                7.7.2              pyhd8ed1ab_0    conda-forge
> jupyter_client            7.4.2              pyhd8ed1ab_0    conda-forge
> jupyter_core              4.11.1          py310h06a4308_0    defaults
> jupyter_server            1.21.0             pyhd8ed1ab_0    conda-forge
> jupyterlab                3.4.8              pyhd8ed1ab_0    conda-forge
> jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
> jupyterlab_server         2.15.2          py310h06a4308_0    defaults
> jupyterlab_widgets        1.1.1              pyhd8ed1ab_0    conda-forge
> tqdm                      4.64.1          py310h06a4308_0    defaults
> widgetsnbextension        3.6.1              pyha770c72_0    conda-forge
> ```
> 
> Downgrading ipywidgets to 7 (8.0.2→7.7.2) also downgraded jupyterlab_widgets (3.0.3→1.1.1) and widgetsnbextension (4.0.3→3.6.1).

Thanks it worked, after executing these commands in Jupyter Notebook and restarting the Kernel!
```
%pip install --force-reinstall -v "ipywidgets == 7.7.2"
%pip install --force-reinstall -v "jupyterlab_widgets == 1.1.1"
```

**Comment by jkmackie** [comment]
End of February 2023, I downgraded ipywidgets from version 8 to version 7 to fix the missing tqdm in Jupyter VSCode.  

- ipywidgets 8.0.0 was released 18 August 2022
- My Jupyter extension is version:   v2023.1.2010391206

Per Don Jayamanne (https://github.com/microsoft/vscode-jupyter/issues/8552#issuecomment-1304892726), there were many, many technical hurdles to get packages like tqdm working in VSCode!

**Opinion**
The single-line ASCII tqdm delivers the required user info.  Moreover, even printing a separate text line--with each progress increment--is preferable to tqdm not working.

My layman's questions:  Are Jupyter extension and IPYWidgets dependencies with tqdm truly necessary to print progress updates?  Should there be fallback behavior?  I hope this feedback is beneficial to the expert developers.  :-)
